data/json/itemgroups/Locations_MapExtras/locations.json

```diff
@@ -87,16 +87,14 @@
     "ammo": 100,
     "subtype": "distribution",
     "entries": [
-      { "group": "guns_pistol_common", "prob": 20 },
```

Comment:
Not going to repeat this over and over, you can see the pattern here where common is more common, rare is more rare, and gun types are heavily skewed pistol > rifle > shotgun.